### DISCUSSION TITLE
Azure PR description max 4000 characters reached

### DISCUSSION THREAD

**Discussion by chrif** [renovatebot/renovate#21880] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.62.0

### If you're self-hosting Renovate, select which platform you are using.

None

### If you're self-hosting Renovate, tell us what version of the platform you run.

Azure DevOps

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Is there a Renovate config to post the body of the pull request in azure platform in a comment of the PR instead of the description of the PR? The description of the PR has a 4000 characters limit easily reached by Renovate grouped PRs, while the comments have no limit. I know I could modify the body template to remove the changelogs (or something else) or have smaller groups, but I really like the body as it is. And I like my groups as they are now.

### What you tried so far.

I didn't try anything yet. chatGPT mentionned this but I think he made it up 😂 
```
{
  "platform": "azure",
  "azure": {
    "platformOptions": {
      "prBodyIsComment": true
    }
  }
}
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
This capability doesn't exist

**Reply by HaoranLI9** [reply]
Hello,
I met the same problem in Bitbucket. The description of "Configure Renovate" shows the information about "Detected Package Files" and "What to Expect". My repo is too big so that it can't show all.
Is there a way can solve it? Like saving the description to a txt file? Renovate has the configuratiob about that?

Thanks a lot
